spec/lib/post_revisor_spec.rb

```diff
@@ -709,6 +709,25 @@
       end
     end
 
+    describe "welcome topic" do
+      before do
+        SiteSetting.welcome_topic_id = topic.id
+      end
+
+      it "should publish welcome topic edit message" do
+        revisor = PostRevisor.new(post)
+        first_post = topic.posts.by_post_number.first
```

Comment:
Looks like we're not using the `first_post` variable in test. May I know why we need it here?

Reply:
Ah, that was because I copied the existing test that was using it. I updated the test along with the old test to use `first_post`.